daemon/mgr/container.go

```diff
@@ -2321,7 +2321,9 @@ func (mgr *ContainerManager) setMountPointDiskQuota(ctx context.Context, c *Cont
 		}
 		err := quota.SetDiskQuota(mp.Source, size, qid)
 		if err != nil {
-			return err
+			// just ignore set disk quota fail
```

Comment:
@rudyfly could we add reason at here, since there is no information about why we ignore the error?

Reply:
Because of the different environment problems, setting disk quota may be failed, in order to start container successfully, so we ignore the errors that cause by diskquota's tools.